cmd/crd-schema-gen/main.go

```diff
@@ -0,0 +1,43 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/pflag"
+
+	crdschema "sigs.k8s.io/controller-tools/pkg/crd-schema"
+	"sigs.k8s.io/controller-tools/pkg/genall"
+)
+
+func main() {
+	apiPackages := pflag.StringSlice("api-packages", nil, "a list of fully-qualified API Golang packages")
```

Comment:
would like to pass `pkg/apis` here and the command should find the versions below. But `pkg/apis` does not work as root because no metav1 import can be found. How are root supposed to work? Are they recursive? Or do I have to list subpackages here and pass all of them as roots?